lib/auth/resetpasswordtoken_test.go

```diff
@@ -163,6 +164,94 @@ func (s *ResetPasswordTokenTest) TestCreateResetPasswordTokenErrors(c *C) {
 
 	for _, tc := range testCases {
 		_, err := s.a.CreateResetPasswordToken(context.TODO(), tc.req)
-		c.Assert(err, NotNil, Commentf("test case %q", tc.desc))
+		c.Assert(err, check.NotNil, check.Commentf("test case %q", tc.desc))
 	}
 }
+
+// TestFormatAccountName makes sure that the OTP account name fallback values
+// are correct.
+func (s *ResetPasswordTokenTest) TestFormatAccountName(c *check.C) {
+	var tests = []struct {
+		inDebugAuth    *debugAuth
+		outAccountName string
+		outError       bool
+	}{
+		// Failed to fetch proxies.
```

Comment:
Add a `desc string` field to the tests struct and print these with `c.Log`.
That way it's easier to figure out which test case failed from output.